src/collections/_documentation/development/contribute/environment.md

```diff
@@ -11,38 +11,114 @@ Follow [the Docker file in the repository](https://github.com/getsentry/sentry/b
+
+One thing that requires manual attention is `docker`, which should have just been installed. Open up Spotlight, search for "Docker" and start it. You should soon see the docker icon in your macOS toolbar. Docker will automatically run on system restarts, so this should be the only time you do this.
+
+You can verify that docker is running by running `docker ps`. If it doesn't error with something like `Error response from daemon: dial unix docker.raw.sock: connect: connection refused`, you're good to continue.
+
+
+### Macintosh OS X - Python
+
+Unfortunately, homebrew recently removed Python 2. While some versions of macOS ship with Python 2, it's recommended to not use the system's installation. Sentry also requires a specific version of Python, as shown in the file `.python-version`, and this may differ from the system's Python.
+
+We recommend using [pyenv](https://github.com/pyenv/pyenv) to install and manage python versions. It should have already been installed earlier when you ran `brew bundle`. But you'll need to make some manual changes to your shell initialization files.
 
-Setup and activate a python 2 virtual environment:
+Make sure your `~/.bash_profile` contains the following:
 
 ```bash
-python2 -m pip install virtualenv
-python2 -m virtualenv .venv
-source .venv/bin/activate
+source ~/.bashrc
+```
+
+And your `~/.bashrc`:
+
+```bash
+eval "$(pyenv init -)"
+```
+
+Now, if you try and run `pyenv`, you'll probably get a command not found. Your shell needs to be reloaded. You can either reload it in-place, or close your terminal and start it again and cd into sentry. To reload it, run:
+
+```bash
+PATH= exec /bin/bash -l
+```
+
+If it worked, you should be able to run `pyenv` and see some help output.
+
+Finally, to install python, run `pyenv install`. This will take a while, since your computer is actually compiling python! To verify everything worked, running `which python` should result in something like `/Users/you/.pyenv/shims/python`.
+
+
+### Macintosh OS X - NodeJS
+
+Sentry also requires a specific version of NodeJS. Like pyenv, we recommend using [volta](https://github.com/volta-cli/volta) to install and manage node versions. Unfortunately, brew doesn't provide volta yet, but installation is quite easy. Run:
+
+```bash
+curl https://get.volta.sh | bash
+```
+
+The volta installer will conveniently make changes to your shell installation files for you, but it's good to verify. Your `~/.bash_profile` should be the same, but make sure your `~/.bashrc` looks like this:
+
+```bash
+eval "$(pyenv init -)"
+
+export VOLTA_HOME="~/.volta"
+grep --silent "$VOLTA_HOME/bin" <<< $PATH || export PATH="$VOLTA_HOME/bin:$PATH"
+```
+
+Again, reload your shell (or restart your terminal, and cd into sentry):
+
+```bash
+PATH= exec /bin/bash -l
+```
+
+Now, if you try and run `volta`, you should see some help text. To install node, run:
+
+```bash
+volta install node@10.16.3
```

Comment:
We should not put the node version in docs as I'm 100% sure we'll diverge.

Reply:
See discussion here: https://github.com/getsentry/sentry-docs/pull/1495#discussion_r377890780